linode/manager
# fix: [M3-7268] - Unassign multiple Linodes from Subnet

## Description 📝
Fix bug where selecting a single Linode in the Unassign Linodes drawer visually selects everything in the dropdown

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/115299789/f60c0de4-2514-4bd4-b004-6abc4353e837" /> | <video src="https://github.com/linode/manager/assets/115299789/a5daa597-0e67-46b1-8478-6a34bfc5b543" /> |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Point to the dev env and ensure your account has vpc customer tags

### Reproduction steps
(How to reproduce the issue, if applicable)
- Ensure you are on the develop branch or the remote dev environment
- Create a VPC and Subnet if you don't already have one
- Go to the VPC's details page and assign some Linodes to the Subnet via the kebab action menu in the table
- Click on Unassign Linodes via the kebab action menu and try to select a Linode
  - You should see that all Linodes are visually selected instead of just the one

### Verification steps 
(How to verify changes)
- Confirm that you are able to select Linodes as normal and then click Unassign Linodes
- Confirm that the Subnet table removes the unassigned Linodes
- Confirm that the inline singular Unassign Linode still works as expected

## As an Author I have considered 🤔

*Check all that apply*
